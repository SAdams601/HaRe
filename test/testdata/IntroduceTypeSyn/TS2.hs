module TS2 where

getInt :: (String, Int) -> Int
getInt (_, i) = i

count :: String -> (Int, String)
count s = (length s, s)

mk :: Int -> String -> (String, Int)
mk i s = (s, i)
