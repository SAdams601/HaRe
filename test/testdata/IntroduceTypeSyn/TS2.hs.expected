module TS2 where

type Foo = (String, Int)

getInt :: Foo -> Int
getInt (_, i) = i

count :: String -> (Int, String)
count s = (length s, s)

mk :: Int -> String -> Foo
mk i s = (s, i)